example/src/components/Table.js

```diff
@@ -1,93 +1,70 @@
     width: 100%;
-    box-sizing: border-box;
-`
-
-const Caption = styled.p`
-    margin: 0px 0px 0px 0px;
-    padding: 8px 0px;
-    width: 100%;
-    font-family: "aktiv-grotesk";
-    font-size: 24px;
-    line-height: 32px;
-    text-align: center;
-    background: #514f4b;
-    color: #fbf9f7;
-    border-top-left-radius: 4px;
-    border-top-right-radius: 4px;
-    box-sizing: border-box;
-`
+    background: #F0EEEB;
+    text-align: left;
+    padding: 8px 0px 8px 0px;
 
-const TableRow = styled.div`
-    margin: 8px 0px 8px 0px;
-    display: flex;
+    & > tbody {
 
-    p:nth-child(1) {
-        min-width: 112px;
-    }
-
-    p:nth-child(2) {
-        min-width: 80px;
-    }
+        & > tr {
+            color: #514F4B;
+            font-family: 'aktiv-grotesk';
 
-    p:nth-child(3) {
-        width: 100%;
+            & > td {
+                /* font-family: 'aktiv-grotesk'; */
+                /* background: blue; */
+            }
+        }
     }
 `
 
-const TableHeader = styled.p`
-    margin: 0px 0px 0px 0px;
-    padding: 8px 0px 8px 8px;
-    text-transform: uppercase;
-    background: #d8d4cf;
-    color: #514f4b;
-    /* border: 1px dotted green; */
-`
+function Table(props) {
 
-const TableData = styled.p`
-    margin: 0px 0px 0px 0px;
-    padding: 0px 0px 0px 8px;
-    /* border: 1px dotted green; */
-`
+    window.onload = init
+    function init() {
+        props.data.templateProps.forEach((availableProp) => {
+            let oddHead = document.getElementById('tbody')
+            let newTR = document.createElement('tr')
+            let td1 = document.createElement('td')
+            let td2 = document.createElement('td')
+            let td3 = document.createElement('td')
+            let td4 = document.createElement('td')
+    
+            let td1Text = document.createTextNode(availableProp.propName)
+            let td2Text = document.createTextNode(availableProp.propType)
+            let td3Text = document.createTextNode(availableProp.options)
+            let td4Text = document.createTextNode(availableProp.propDescription)
 
-function Table(props) {
-    // console.log(props)
-    let componentDetails = decideInfo(props.component)
-    const { availableProps } = componentDetails
+            td1.appendChild(td1Text)
+            td2.appendChild(td2Text)
+            td3.appendChild(td3Text)
+            td4.appendChild(td4Text)
 
-    let tableRows = availableProps.map((propObj, idx) => {
-        const { propName, propType, propDescription } = propObj
-        return (
-            <TableRow key={idx}>
-                <TableData>{propName}</TableData>
-                <TableData>{propType}</TableData>
-                <TableData>{propDescription}</TableData>
-            </TableRow>
-        )
-    })
+            newTR.appendChild(td1)
+            newTR.appendChild(td2)
+            newTR.appendChild(td3)
+            newTR.appendChild(td4)
+            oddHead.appendChild(newTR)
+        })
+    }
```

Comment:
This feels _way_ too imperative. I'll poke around to see if I can find a more "React" way of doing this, but it feels like it's side-stepping the render cycle by imperatively adding elements to the DOM, listening for window load, etc.